docs/source/VUDRM-token.md

```diff
@@ -0,0 +1,157 @@
+| `duration_rental`  | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay rental licenses.                                                               |
+| `duration_lease`   | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay lease licenses.                                                              |
+| `duration_persist` | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay persist licenses.                                                           |
+
+This table is not an exhaustive list, for example it does not include advanced PlayReady settings. If you require the use of more advanced settings please contact support@vualto.com
+
+The `polbegin` and `polend` settings use the timezone set at the account level. Please contact support@vualto.com to confirm the timezone for your account. 
+
+There are also limitations depending on environments that are not explained in the table. Please refer to the following sections for more detail:
+
+### PlayReady DRM policy
+
+PlayReady has an extensive list of policy options described [here](https://docs.microsoft.com/en-us/playready/overview/license-and-policies). The majority of these options are supported via the VUDRM token's policy. Please contact support@vualto.com for more details.
+
+### Fairplay DRM policy
+
+Fairplay has three types of license; `rental`, `lease`, and `persist`. 
+
+Fairplay licenses can only be persisted past the user session by an offline enabled iOS application. When using Safari a session will be preserved until the browser tab is closed.
+
+Fairplay does not allow playback over non-HDCP connections. 
+
+#### Fairplay rental DRM policy
+
+You can specify a `rental` license by setting the `type` key to `r` in the DRM policy.
+
+You can set the expiry of the license using the `duration_rental` key or the `polend` key. If both `duration_rental` and `polend` are set in a policy then `duration_rental` will be used.
+
+When using the `type` of `rental` playback will continue after license expiry until the encryption keys change.
+
+#### Fairplay lease DRM policy
+
+You can specify a `lease` license by setting the `type` key to `l` in the DRM policy.
+
+You can set the expiry of the license using the `duration_lease` key or the `polend` key. If both `duration_lease` and `polend` are set in a policy then `duration_lease` will be used.
+
+Using the `type` of `lease` will cause playback to stop at the license expiry. Normally a player will make a new license request at this point, please ensure the VUDRM token is updated before further license requests are made. Using an expired VUDRM token will cause the license request to fail.
+
+#### Fairplay persist DRM policy
+
+A `persist` license is used for offline playback. 
+
+Setting `licache` to `yes` or the `type` to `p` will generate a persisted license. Setting `licache` to `yes` will override any other `type` settings.
```

Comment:
Is it licache or liccache as in the table above. One or other is wrong - Or this needs serious highlighting if both are possible!